dotCMS/src/main/java/com/dotcms/rest/api/v1/system/monitor/MonitorResource.java

```diff
@@ -0,0 +1,343 @@
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.util.*;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.index.query.QueryBuilders;
+import com.dotcms.repackage.javax.ws.rs.*;
+import com.dotcms.content.elasticsearch.business.IndiciesAPI.IndiciesInfo;
+import com.dotcms.content.elasticsearch.util.ESClient;
+import com.dotcms.repackage.javax.ws.rs.core.Context;
+import com.dotcms.repackage.javax.ws.rs.core.MediaType;
+import com.dotcms.repackage.javax.ws.rs.core.Response;
+import com.dotcms.repackage.javax.ws.rs.core.Response.ResponseBuilder;
+import com.dotcms.repackage.org.glassfish.jersey.server.JSONP;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.util.json.JSONObject;
+
+import net.jodah.failsafe.CircuitBreaker;
+import net.jodah.failsafe.Failsafe;
+
+/**
+ * 
+ * 
+ * Call
+ *
+ */
+@Path("/v1/system-status")
+public class MonitorResource {
+
+    static private long LOCAL_FS_TIMEOUT=1000;
+    static private long CACHE_TIMEOUT=1000;
+    static private long ASSET_FS_TIMEOUT=1000;
+    static private long INDEX_TIMEOUT=1000;
+    static private long DB_TIMEOUT=1000;
+    private ExecutorService executorService = Executors.newCachedThreadPool();
+
+    @NoCache
+    @GET
+    @JSONP
+    @Path("/")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response test(final @Context HttpServletRequest request) throws Throwable {
+        // force authentication
+        //InitDataObject auth = webResource.init(false, httpRequest, false);  cannot require as we cannot assume db or other subsystems are functioning
+        boolean extendedFormat = false;
+        if (request.getQueryString() != null && "extended".equals(request.getQueryString())) {
+            extendedFormat = true;
+        }
+
+        final String config_ip_acl = Config.getStringProperty("SYSTEM_STATUS_API_IP_ACL", "127.0.0.1/32,0:0:0:0:0:0:0:1/128");
+        String[] aclIPs = null;
+        if(config_ip_acl != null)
+            aclIPs = config_ip_acl.split(",");
+
+        String clientIP = HttpRequestDataUtil.getIpAddress(request).toString().split("/")[1];
+        Boolean accessAllowed = false;
+        if(aclIPs == null) {
+            accessAllowed = true;
+        }
+        else {
+            for(String aclIP : aclIPs) {
+                if(IPUtils.isIpInCIDR(clientIP, aclIP)){
+                    accessAllowed = true;
+                    break;
+                }
+            }
+        }
+
+        try {
+            ResponseBuilder builder = null;
+            if (accessAllowed) {
+                IndiciesInfo idxs = APILocator.getIndiciesAPI().loadIndicies();
+                LOCAL_FS_TIMEOUT = Config.getLongProperty("SYSTEM_STATUS_API_LOCAL_FS_TIMEOUT", 1000);
```

Comment:
these don't look as a constant, so should be not following the upper-case pattern, however if they have to be a class attribute they should be atomic or volatile